Gemfile

```diff
@@ -41,7 +42,7 @@ gem "tzinfo-data", platforms: %i[ mingw mswin x64_mingw jruby ]
 
 # Reduces boot times through caching; required in config/boot.rb
 gem "bootsnap", require: false
-
+gem 'devise'
```

Comment:
Is this going to be a full-stack app (with Rails views) or just an API?

If it's an API, you might have to consider using another gem that provides auth tokens instead of storing session in cookies. I think the one we use more is https://github.com/lynndylanhurley/devise_token_auth